client/src/pages/Resources.js

```diff
@@ -118,6 +118,13 @@ function Resources(props) {
       );
     }
 
+    newResources.result.sort(function(a, b) {
```

Comment:
maybe also good to put the sort function in `util` or something so that we dont have to repeat code